prusti-common/src/vir/low_to_viper/domain.rs

```diff
@@ -35,20 +35,29 @@ impl<'a, 'v> ToViper<'v, viper::DomainFunc<'v>> for (&'a String, &'a DomainFunct
     }
 }
 
-impl<'a, 'v> ToViper<'v, Vec<viper::NamedDomainAxiom<'v>>>
+impl<'a, 'v> ToViper<'v, Vec<viper::AnonymousDomainAxiom<'v>>>
```

Comment:
I see how generating names can be annoying when experimenting, but I find having unique names useful in my encodings, where I have hundreds of axioms. Therefore, I would prefer to revert this at least for `low_to_viper`.